src/services/platform/notification-adapter/notification.adapter.ts

```diff
@@ -0,0 +1,245 @@
+import { Inject, Injectable, LoggerService } from '@nestjs/common';
+import { WINSTON_MODULE_NEST_PROVIDER } from 'nest-winston';
+import { LogContext } from '@common/enums/logging.context';
+import { NotificationInputAspectCreated } from './dto/notification.dto.input.aspect.created';
+import { NotificationInputCalloutPublished } from './dto/notification.dto.input.callout.published';
+import { NotificationInputAspectComment } from './dto/notification.dto.input.aspect.comment';
+import { NotificationPayloadBuilder } from './notification.payload.builder';
+import { NOTIFICATIONS_SERVICE } from '@common/constants/providers';
+import { ClientProxy } from '@nestjs/microservices';
+import { NotificationEventType } from '@common/enums/notification.event.type';
+import { NotificationInputCollaborationInterest } from './dto/notification.dto.input.collaboration.interest';
+import { NotificationInputUpdateSent } from './dto/notification.dto.input.update.sent';
+import { NotificationInputDiscussionCreated } from './dto/notification.dto.input.discussion.created';
+import { NotificationInputCommunityApplication } from './dto/notification.dto.input.community.application';
+import { NotificationInputCommunityNewMember } from './dto/notification.dto.input.community.new.member';
+import { NotificationInputCommunityContextReview } from './dto/notification.dto.input.community.context.review';
+import { NotificationInputUserRegistered } from './dto/notification.dto.input.user.registered';
+
+@Injectable()
+export class NotificationAdapter {
+  constructor(
+    @Inject(WINSTON_MODULE_NEST_PROVIDER)
+    private readonly logger: LoggerService,
+    private notificationPayloadBuilder: NotificationPayloadBuilder,
+    @Inject(NOTIFICATIONS_SERVICE) private notificationsClient: ClientProxy
+  ) {}
+
+  async calloutPublished(
+    eventData: NotificationInputCalloutPublished
+  ): Promise<boolean> {
```

Comment:
when does it return `false`?
Maybe `void` is more suitable when the same result is always returned.

Reply:
updated to be void